views/about.twig

```diff
@@ -85,21 +85,21 @@
             <li><a href="https://twitter.com/dave1010">Dave Hulbert / @dave1010</a></li>
             <li><a href="https://twitter.com/rossey">Alex Ross / @rossey</a></li>
             <li><a href="https://twitter.com/hughbertd">Hugh Downer / @HughbertD</a></li>
-            <li><a href="https://twitter.com/fully_baked">Dave Baker / @fully_baked</a></li>
+            <li><a href="https://twitter.com/tweetingsherry">Chris Sherry / @tweetingsherry</a></li>
         </ul>
     </div>
     <div class="wrapper clearfix document">
         <h3><a name="meetings">Meetings</a></h3>
 
-        <p>We meet once a month at <a href="https://www.google.co.uk/maps/place/St+Ambrose+Church/@50.7176004,-1.9018998,17.11z/data=!4m2!3m1!1s0x4873a1a57458cc11:0xf5803aef30b9ec8b">St. Ambrose Church hall in Westbourne</a>,
-        usually on the first Monday (although we shift to the second Monday for bank holidays) from 7.30pm.
-        There are typically two talks and then we go to the pub. There’s always tea and coffee at St. Ambrose, and usually there’ll be pizza; beer is limited to the pub afterwards.</p>
+        <p>We meet once a month at <a href="https://www.google.com/maps?ll=50.724055,-1.906312&z=16&t=m&hl=en-US&gl=US&mapclient=embed&cid=4020339563874231918">the Barclays Eagle Labs in Westbourne</a>,
+        usually on the first Monday (although we shift to the second Monday for bank holidays) from 6.30pm.
+        There are typically one or two talks and then we go to the pub. There’s tea and coffee and occasionally beer available, we provide vegetarian and meat pizzas and can order gluten free on request;</p>
```

Comment:
replace pub with The Libertine and make The Libertine a URL to a map, do the same for below